tests/request/test_build_request.py

```diff
@@ -135,18 +138,47 @@ def test_build_request(body: Dict[str, Any], condition: Expression) -> None:
             "tenant_ids": {"organization_id": 1, "referrer": "test"},
         },
         {"organization_id": 1, "referrer": "test"},
+        "my_request",
+        0,
+        id="multiple projects, no project tenant",
```

Comment:
ids of the tests are the same, change it to make it accurate to what it is testing